src/Tizen.NUI.Components/Theme/ThemeActionDataSet.cs

```diff
@@ -0,0 +1,110 @@
+            radioButtonActions.AddSelectorAction<float?>(nameof(RadioButton), nameof(ButtonStyle.Text), nameof(TextLabelStyle.PixelSize));
+            actionMap.Add(radioButtonActions);
+
+
+            var sliderActions = new ThemeActionData(typeof(Slider).FullName, typeof(SliderStyle));
+            sliderActions.AddSelectorAction<Color>(nameof(Slider), nameof(SliderStyle.Track), nameof(ViewStyle.BackgroundColor));
+            sliderActions.AddSelectorAction<Color>(nameof(Slider), nameof(SliderStyle.Progress), nameof(ViewStyle.BackgroundColor));
+            sliderActions.AddSelectorAction<string>(nameof(Slider), nameof(SliderStyle.Thumb), nameof(ImageViewStyle.ResourceUrl));
+            sliderActions.AddSelectorAction<string>(nameof(Slider), nameof(SliderStyle.Thumb), nameof(ViewStyle.BackgroundImage));
+            sliderActions.AddSelectorAction<string>(nameof(Slider), nameof(SliderStyle.ValueIndicatorImage), nameof(ImageViewStyle.ResourceUrl));
+            actionMap.Add(sliderActions);
+
+
+            var switchActions = new ThemeActionData(typeof(Switch).FullName, typeof(SwitchStyle));
+            switchActions.AddSelectorAction<string>(nameof(Switch), nameof(SwitchStyle.Track), nameof(ImageViewStyle.ResourceUrl));
+            switchActions.AddSelectorAction<string>(nameof(Switch), nameof(SwitchStyle.Thumb), nameof(ImageViewStyle.ResourceUrl));
+            switchActions.AddSelectorAction<Color>(nameof(Switch), nameof(SwitchStyle.Text), nameof(TextLabelStyle.TextColor));
+            switchActions.AddSelectorAction<float?>(nameof(Switch), nameof(SwitchStyle.Text), nameof(TextLabelStyle.PixelSize));
+            actionMap.Add(switchActions);
+
+            var paginationActions = new ThemeActionData(typeof(Pagination).FullName, typeof(PaginationStyle));
+            paginationActions.AddSelectorAction<string>(nameof(Pagination), nameof(PaginationStyle.IndicatorImageUrl));
+            actionMap.Add(paginationActions);
+
+            var scrollbarActions = new ThemeActionData(typeof(Scrollbar).FullName, typeof(ScrollbarStyle));
+            scrollbarActions.AddAction<Color>(nameof(Scrollbar), nameof(ScrollbarStyle.TrackColor));
+            scrollbarActions.AddAction<Color>(nameof(Scrollbar), nameof(ScrollbarStyle.ThumbColor));
+            actionMap.Add(scrollbarActions);
+
+            var recyclerViewItemActions = new ThemeActionData(typeof(RecyclerViewItem).FullName, typeof(RecyclerViewItemStyle));
+            recyclerViewItemActions.AddSelectorAction<Color>(nameof(RecyclerViewItem), nameof(ViewStyle.BackgroundColor));
+            actionMap.Add(recyclerViewItemActions);
+
+            var defaultTitleItemActions = new ThemeActionData(typeof(DefaultTitleItem).FullName, typeof(DefaultTitleItemStyle));
+            defaultTitleItemActions.AddSelectorAction<Color>(nameof(DefaultTitleItem), nameof(ViewStyle.BackgroundColor));
+            actionMap.Add(defaultTitleItemActions);
+
+            return actionMap;
+        }
+    }
+}
```

Comment:
Does this work affect launching performance?

Reply:
No. This code is not run when launching.
Tizen-theme-manager's ThemeChanged event could trigger this.